src/group_theory/subgroup/simple.lean

```diff
@@ -0,0 +1,63 @@
+/-
+Copyright (c) 2021 Aaron Anderson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Aaron Anderson
+-/
+
+import group_theory.subgroup.actions
+
+/-!
+# Simple groups
```

Comment:
Can you remove the documentation for `is_simple_group` from `subgroup/basic.lean` to here?

Reply:
Thanks, done.